src/raster/r.bioclim/r.bioclim.html

```diff
@@ -40,6 +40,14 @@ <h4>Quarter years</h4>
 href="http://worldclim.org/bioclim">Worldclim</a> were computed. This 
 option should be used when long-term averages are used as input.
 
+<h4>Precipitation data</h4>
+Check the unit of measure of precipitation data. To calculate the bio18 and bio19
+r.bioclim rounds the value of the raster to the closest integer. If the unit
+of measure of your data is, for example, kg m-2 m-1 (eg fluxes per second) as in
+CHELSA data <a href="<a href="https://chelsa-climate.org/">CHELSA</a>"></a>
+you will have a raster of 0s for the bio18 and bio19, because the precipitation values
```

Comment:
```suggestion
<b>r.bioclim</b> rounds the value of the raster to the closest integer value. If the unit
of measure of your data is, for example, kg m-2 m-1 (i.e., fluxes per second) as in
<a href="https://chelsa-climate.org/">CHELSA</a> data you will gain a raster map with
zero values for the bio18 and bio19, because the precipitation values
```

Reply:
Done, thanks